dashboard/client/index.js

```diff
@@ -41,8 +41,18 @@ Template.dashboard.onCreated(function () {
     }
   }
 
-  instance.getAggrData = opts => new Promise((resolve, reject) => {
-    Meteor.call('getAggr', opts, (err, res) => {
+  instance.getChartData = opts => new Promise((resolve, reject) => {
+    Meteor.call('getAggregatedData', opts, (err, res) => {
+      if (err) reject(err)
+      Meteor.call('parseDataForNvd', res, (err1, res1) => {
+        if (err1) reject(err1)
+        resolve(res1)
+      })
+    })
+  })
+
+  instance.parseDataForNvd = data => new Promise((resolve, reject) => {
+    Meteor.call('parseDataForNvd', data, (err, res) => {
```

Comment:
Again, it seems like this parsing step (preparing it for NVD3 should either:

- happen as a sub-function call in the `getChartData` server method
- happen as a client side parsing function

It seems inefficient that we would get data from the server and send it back to the server for further parsing.